tutorials/deploy-multitenant-app-kyma/deploy-multitenant-app-kyma.md

```diff
@@ -0,0 +1,178 @@
+```
+
+    In the directory `kyma-multitenant-node`, build the image for the approuter app from source, for example:
+```Shell / Bash
+pack build multitenant-kyma-backend --builder paketobuildpacks/builder:full
+docker tag multitenant-kyma-backend <docker-hub-account>/multitenant-kyma-backend:v1
+```
+
+
+
+[DONE]
+[ACCORDION-END]
+
+[ACCORDION-BEGIN [Step 4: ](Push OCI Image to Docker Hub)]
+
+**1.** Log in to Docker using this command:
+
+```Shell / Bash
+docker login -u <docker-id> -p <password>
+```
+
+**2.** Push the local images into the Docker Hub:
+
+```Shell / Bash
+docker push <docker-hub-account>/multitenant-approuter:v1
+docker push <docker-hub-account>/multitenant-kyma-backend:v1
+```
+
+>  For more details, see the [Kubernetes documentation](https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/).
+
+
+
+
+[DONE]
+[ACCORDION-END]
+
+[ACCORDION-BEGIN [Step 5: ](Create Namespace)]
+
+If you followed the tutorials [Create a Basic Node.js Application with Express Generator](basic-nodejs-application-create) and [Deploy a Node.js Application in the Kyma Runtime](deploy-nodejs-application-kyma), you have created a namespace in the Kyma environment called `multitenancy-ns`. If not, create a new namespace through the Kyma dashboard or `kubectl` CLI, for example, called `multitenancy-ns`:
+
+!![image-20220214150615225](image-20220214150615225.png)
+
+
+
+
+[DONE]
+[ACCORDION-END]
+
+[ACCORDION-BEGIN [Step 6: ](Deploy Secret for Docker Hub)]
+
+Since the OCI image is stored in your Docker hub, you need to provide the access information to your Kyma cluster that you can pull the images from those repositories.
+
+If you followed the tutorials [Create a Basic Node.js Application with Express Generator](basic-nodejs-application-create) and [Deploy a Node.js Application in the Kyma Runtime](deploy-nodejs-application-kyma), you have configured the credential of your Docker Hub as a Secret. If not, create a new Secret with the following command, replace the placeholder values according to your account:
+
+```Shell / Bash
+kubectl -n multitenancy-ns create secret docker-registry registry-secret --docker-server=https://index.docker.io/v1/  --docker-username=<docker-id> --docker-password=<password> --docker-email=<email>
+```
+
+
+Therefore, all deployment files contain an `imagePullSecret` entry, which should be set to `registry-secret`.
+
+```YAML
+imagePullSecrets:
+        - name: registry-secret # replace with your own registry secret
+```
+
+
+
+
+[DONE]
+[ACCORDION-END]
+
+[ACCORDION-BEGIN [Step 7: ](Deploy Application into Kyma Runtime)]
+
+**1.** Deploy consumed services by executing this command:
+
+```Shell / Bash
+kubectl -n multitenancy-ns apply -f k8s-deployment-services.yaml
+```
+
+**2.** Deploy the approuter applications by executing this command:
+
+```Shell / Bash
+kubectl -n multitenancy-ns apply -f k8s-deployment-approuter.yaml
+```
+
+**3.** Deploy the backend applications by executing this command:
```

Comment:
Please add indentations

Reply:
> Please add indentations

Yes, these are all the cases that we discussed in the email. 
For this case (order list item followed by code snippet), if I tried to indent by removing the blank line, it will hit an error like this. Actually, I’ve found this kind of error before, but I can only esit as the workaround you can see
![image001](https://user-images.githubusercontent.com/99162172/162387173-0b98b6f4-1a3e-447d-99f5-03cf6e5e857f.png)
Considering our purpose of improving readability, I think this workaround is enough, let’s make it as ordered list but without indenting, how do you think of it?

Reply:
The final readability is good I think:
[Tutorial] Introduce Multitenant Application Development on SAP BTP, Kyma Runtime: https://developers-qa.sap.com/tutorials/introduce-multitenant-application-development-kyma.html
[Tutorial] Create and Configure the Approuter Application for a Multitenant Application: https://developers-qa.sap.com/tutorials/create-configure-approuter-multitenant-application.html
[Tutorial] Secure a Multitenant Application with the Authorization and Trust Management Service (XSUAA) in the Kyma Runtime: https://developers-qa.sap.com/tutorials/secure-multitenant-app-xsuaa-kyma.html
[Tutorial] Implement Subscription callback APIs for a Multitenant Application: https://developers-qa.sap.com/tutorials/implement-subscription-callback-multitenant.html
[Tutorial] Register a Multitenant Application to the SAP SaaS Provisioning Service: https://developers-qa.sap.com/tutorials/register-multitenant-app-saas-provisioning-service.html
[Tutorial] Deploy a Multitenant Application to a Provider subaccount, Kyma Runtime: https://developers-qa.sap.com/tutorials/deploy-multitenant-app-kyma.html
[Tutorial] Subscribe a Multitenant Application by a Consumer: https://developers-qa.sap.com/tutorials/subscribe-multitenant-app-customer.html

Reply:
> Please add indentations

Of course, if we can fix the error above, I will add official identation.